packages/account-v2/src/components/ErrorMessage/ErrorMessage.tsx

```diff
@@ -6,19 +6,19 @@ type TErrorMessage = {
 };
 
 export const ErrorMessage = ({ message }: TErrorMessage) => (
-    <div className='grid justify-center w-full justify-items-center mt-4000 gap-1000'>
+    <div className='grid justify-center w-full justify-items-center mt-[80px] gap-20'>
```

Comment:
@thisyahlen-deriv , is there any plan to add the missing measurements like `80px` to tailwind config?

Reply:
if yall need it sure man why not, we can put the cap at 100px.

Reply:
Added the measurements in our tailwind config 😎